openshift/route.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  labels:
+    app.kubernetes.io/name: bf2-arch-bot
+  name: bf2-arch-bot
+spec:
+  port:
+    targetPort: http
+  tls:
```

Comment:
Why are we using port 80 if we're using TLS? Wouldn't 443 would be more correct?

Reply:
There are a couple of layers here:
* The Quarkus process in the container is listening on port 8080, and that's exposed in the Pod definition
* The Service is listening on port 80, and sending any connections from there to the Pod (service is like an internal load balancer that provides stable name/IP, which pods don't have, since they can be replaced)
* The Route is listening on 443, (and TLS termination happens here, so the Pod doesn't need to know about certs, etc), and sending connections to port 80 on the Service, which end up on port 8080 of the Pod.

So it's HTTPS as far as the HAProxy ingress controller, then HTTP inside the cluster.